ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md

```diff
@@ -434,40 +468,101 @@ Below you will find two examples of documentation automatically created by Ansib
 
 ## How do I use AVD?
 
-AVD comes with pre-built templates that you can either use as-is or adapt to your liking. For example
-[ansible-avd-cloudvision-demo](https://github.com/arista-netdevops-community/ansible-avd-cloudvision-demo) is a 3-stage leaf spine fabric
-that can be deployed CloudVision.
+AVD comes with pre-built templates that you can either use as-is or adapt to your liking.
 
 Once the templates reflect your desired network configuration, you deploy the configuration either directly to the Arista EOS devices or to
-Configlets within CloudVision. This is typically executed from the CLI - for example:
+Configlets within CloudVision. This is typically executed from the CLI - for example from your Ansible AVD Examples directory. 
+
+### Using Ansible AVD with direct eAPI connectivity to the switches
+
+If you want to push to switches in the entire FABRIC using eAPI and your playbook looks as follows:
+
+```yml
+---
+- name: Run AVD
+  hosts: FABRIC
+  gather_facts: false
+  tasks:
+    - name: generate intended variables
+      import_role:
+        name: arista.avd.eos_designs
+    - name: generate device intended config and documentation
+      import_role:
+        name: arista.avd.eos_cli_config_gen
+    - name: deploy configuration to device
+      import_role:
+         name: arista.avd.eos_config_deploy_eapi
+```
+
+You would execute it using the following command:
 
 ```shell
-ansible-playbook playbooks/dc1-fabric-deploy-cvp.yml --tags build
+ansible-playbook playbook.yml
```

Comment:
this assumes that the playbook file is called `playbook.yml` which may not be the case

Reply:
Added more clarity